.github/PULL_REQUEST_TEMPLATE

```diff
@@ -20,22 +12,13 @@ Add mentions of things that are not covered here and are planed to be done in se
 
 ## Contributor Checklist:
 
-* [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://docs.twistedmatrix.com/en/stable/core/development/policy/coding-standard.html)
-* [ ] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://github.com/twisted/trac-wiki-archive/blob/trunk//ReviewProcess.mediawiki#News_files))
-* [ ] The title of the PR starts with the associated issue number (without the `#` character).
-* [ ] I have updated the automated tests and checked that all checks for the PR are green.
-* [ ] The merge commit will use the below format
-  The first line is automatically generated by GitHub based on PR ID and branch name.
-  The other lines generated by GitHub should be replaced.
-
-```
-Merge pull request #123 from twisted/4356-branch-name-with-issue-id
-
-Author: <github_username>, <github_usernames_if_more_authors>
-Reviewer: <github_username>, <github_username_if_more_reviewers>
-Fixes #<github_issue_number>
-Fixes #<another_if_more_in_one_PR>
-
-Long description providing a summary of these changes.
-(as long as you wish)
-```
+This process apply to *all* pull requests, no matter how small.
+Have a look at [our developer documentation](https://docs.twisted.org/en/latest/core/development/dev-process.html) before submitting your Pull Request.
+
+Below is a non-exaustive list (as a reminder):
+
+* The title of the PR should describe the changes and starts with the associated issue number (including the `#` character for GitHub auto-links).
+* A release notes news fragment file was create in src/twisted/newsfragments/ (see: [Release notes fragments docs.](https://docs.twisted.org/en/latest/core/development/dev-process.html#release-notes-management))
+* The automated tests were updated.
+* Once all checks are green, request a review by leaving a comment that contains the `please review` words.
+  This will trigger the review process.
```

Comment:
I'd say what this process looks like so people can evaluate whether it is happening or not (and if not complain to someone).

I'm guessing it is something like "a bot requests a review from some twisted-committers team"?

Reply:
True. Thanks.